packages/expo/tsconfig.json

```
@@ -3,8 +3,7 @@
 {
   "extends": "expo-module-scripts/tsconfig.base",
   "compilerOptions": {
-    "outDir": "./build",
-    "declaration": false
+    "outDir": "./build"
```

```suggestion
    "outDir": "./build",
    "declaration": false
```

See this commit https://github.com/expo/expo/commit/799ace52e88aa0c9df6c0d0ea99672252fa60e79 🙂